src/final2/subtests/InvalidCommandTest.java

```diff
@@ -0,0 +1,942 @@
+package final2.subtests;
+
+import org.junit.Test;
+
+import test.Input;
+import test.runs.ErrorRun;
+import test.runs.NoOutputRun;
+import test.runs.Run;
+
+/**
+ * Tests the program for error messages with not well-formed or invalid commands.
+ * 
+ * @author Roman Langrehr
+ * @since 26.03.2015
+ *
+ */
+public class InvalidCommandTest extends LangtonSubtest {
+
+	private static final String[] ARGUMENT_COMMANDS = new String[] {
+			"create",
+			"field",
+			"position",
+			"direction",
+			"escape",
+			"move"
+	};
+
+	private static final String[] INVALID_ANT_NAMES = new String[] {
+			"ab",
+			"�",
+			"�",

```

Comment:
It appears you still do not use UTF-8 (see https://github.com/jGleitz/JUnit-KIT/pull/122#issuecomment-86487125).
However, we should probably use backslash escape here for these two special symbols, in case others compile our tests with a non-UTF-8 encoding.